signer_server.go

```diff
@@ -0,0 +1,224 @@
+// Copyright (C) 2013-2017 The btcsuite developers
+// Copyright (C) 2015-2016 The Decred developers
+// Copyright (C) 2015-2022 Lightning Labs and The Lightning Network Developers
+// Copyright (C) 2022 Bottlepay and The Lightning Network Developers
+
+package lndsigner
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/bottlepay/lndsigner/keyring"
+	"github.com/bottlepay/lndsigner/proto"
+	"github.com/bottlepay/lndsigner/vault"
+	"github.com/btcsuite/btcd/btcec/v2"
+	"github.com/btcsuite/btcd/btcec/v2/schnorr"
+	"gopkg.in/macaroon-bakery.v2/bakery"
+)
+
+var (
+	// signerPermissions maps RPC calls to the permissions they require.
+	signerPermissions = map[string][]bakery.Op{
```

Comment:
Can be removed?

Reply:
Fixed here and in other places. :heavy_check_mark: